Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -126,14 +128,14 @@ bool VFS::mount_root(FS& file_system)
     }
 
     m_root_inode = move(root_inode);
-    char device_name[32];
-    if (m_root_inode->fs().is_disk_backed()) {
-        auto& device = static_cast<DiskBackedFS&>(m_root_inode->fs()).device();
-        sprintf(device_name, "%d,%d", device.major(), device.minor());
+    char filename[FILENAME_MAX];
+    if (m_root_inode->fs().is_file_backed()) {
+        auto& file = static_cast<FileBackedFS&>(m_root_inode->fs()).file();
+        sprintf(filename, "%s", file.absolute_path(static_cast<FileBackedFS&>(m_root_inode->fs()).file_description()).characters());
     } else {
-        sprintf(device_name, "not-a-disk");
+        sprintf(filename, "not-a-disk");
     }
-    klog() << "VFS: mounted root on " << m_root_inode->fs().class_name() << " (" << device_name << ")";
+    klog() << "VFS: mounted root on " << m_root_inode->fs().class_name() << " (" << filename << ")";
```

Comment:
And you don't even need it; just use the string returned by `static_cast<FileBackedFS&>(m_root_inode->fs()).file_description().absolute_description()` directly.